litellm/caching.py

```diff
@@ -1721,9 +1745,12 @@ def __init__(
         s3_config: Optional[Any] = None,
         s3_path: Optional[str] = None,
         redis_semantic_cache_use_async=False,
+        redis_gptcache_use_async=False,
```

Comment:
no need to add extra flags for gptcache, you can use value of redis_semantic cache flags in order to specify use_async behaviour or define embedding model. If you add more flags, then their mapping at every place will also be required.